src/main/content/instance.html

```diff
@@ -12,15 +14,163 @@
 ---
 {% assign t = site.data[site.active_lang] %}
 <!-- Welcome header -->
-<div id="meta-data" class="container-fluid">
-    <!-- Meta Data is dynamically generated -->
-    <div class="row">
-        <div id="instance-data-container" class="col-md">
-            <h2>{{t.instance.kab-instances}}</h2>
+<div class="bx--grid bx--grid--full-width">
+  <div class="bx--row">
+    <div class="bx--col">
+      <h1>{{t.instance.instances}}</h1>
+    </div>
+  </div>
+  <div class="bx--row">
+    <div class="bx--col-sm-4 bx--col-md-8 bx--col-xlg-4">
+        <ul data-accordion class="bx--accordion" id="instance-accordion">
+            <li data-accordion-item class="bx--accordion__item loading-row">
+              <button class="bx--accordion__heading" aria-expanded="false" aria-controls="paneTest">
+                <div class="bx--accordion__title">
+                  <div data-inline-loading class="bx--inline-loading" role="alert" aria-live="assertive">
+                    <div class="bx--inline-loading__animation">
+                      <div data-inline-loading-spinner class="bx--loading bx--loading--small">
+                        <svg class="bx--loading__svg" viewBox="-75 -75 150 150">
+                          <circle class="bx--loading__background" cx="0" cy="0" r="26.8125" />
+                          <circle class="bx--loading__stroke" cx="0" cy="0" r="26.8125"/>
+                        </svg>
+                      </div>
+                      <svg focusable="false" preserveAspectRatio="xMidYMid meet" style="will-change: transform;" xmlns="http://www.w3.org/2000/svg" class="bx--inline-loading__checkmark-container" hidden="" data-inline-loading-finished="" width="16" height="16" viewBox="0 0 16 16" aria-hidden="true"><path d="M8,1C4.1,1,1,4.1,1,8c0,3.9,3.1,7,7,7s7-3.1,7-7C15,4.1,11.9,1,8,1z M7,11L4.3,8.3l0.9-0.8L7,9.3l4-3.9l0.9,0.8L7,11z"></path><path d="M7,11L4.3,8.3l0.9-0.8L7,9.3l4-3.9l0.9,0.8L7,11z" data-icon-path="inner-path" opacity="0"></path></svg>
+                      <svg focusable="false" preserveAspectRatio="xMidYMid meet" style="will-change: transform;" xmlns="http://www.w3.org/2000/svg" class="bx--inline-loading--error" hidden="" data-inline-loading-error="" width="20" height="20" viewBox="0 0 32 32" aria-hidden="true"><path d="M2,16H2A14,14,0,1,0,16,2,14,14,0,0,0,2,16Zm23.15,7.75L8.25,6.85a12,12,0,0,1,16.9,16.9ZM8.24,25.16A12,12,0,0,1,6.84,8.27L23.73,25.16a12,12,0,0,1-15.49,0Z"></path></svg>
+                    </div>
+                  </div>
+                </div>
+              </button>
+            </li>
+        </ul>
+    </div>
+    <div class="bx--col-sm-4 bx--col-md-4 bx--col-xlg-4 bx--tile-container">
+      <div class="bx--tile">
+        <div class="bx--row instance-card">
+          <div class="bx--col" id="collections-card">
+            <div class="bx--row loading-row">
+              <div class="bx--col">
```

Comment:
you duplicate row and col twice in a row, I would double check you really need that amount of depth.